README.md

```diff
@@ -49,6 +52,10 @@ What pisek verifies (roughly in order):
     - They don't contain unprintable characters
     - They have a newline at the end
     - In Kasiopea mode the files are reasonably small
+ - The task is complete
+    - In CMS there are no redundant lines in the judge's stdout/stderr
```

Comment:
I noticed one more issue. For consistency with lines 41 and 54, we should use this:

```suggestion
    - In CMS mode there are no redundant lines in the judge's stdout/stderr
```

Reply:
This should be better. Mostly because CMS mode is an illusion :D

Reply:
Kasiopea mode is also mentioned on line 41.